rom/laghos_rom.hpp

```diff
@@ -81,6 +120,19 @@ struct ROM_Options
 
     std::string *basename = NULL;
 
+    std::string basisIdentifier = "";
+    std::string greedyParam = "bef";
+    double greedyTol = 0.1; // error indicator tolerance for the greedy algorithm
```

Comment:
In GREEDY.md this is described as a relative tolerance, and my understanding is that errors in Laghos used by the libROM greedy algorithm are always relative errors. It may be clearer to add to the comment here that this is for relative error.